src/NumericalAlgorithms/SpatialDiscretization/OptionTags.hpp

```diff
@@ -11,10 +11,20 @@ namespace SpatialDiscretization::OptionTags {
 /*!
  * \brief Group holding all the options for the spatial discretization of the
  * PDE system.
+ *
+ * For example, when using a discontinuous Galerkin scheme choices like which
```

Comment:
suggest a clarifying rephrase: `...scheme choices like which basis function to use are...` -> `...scheme, options like the basis function choice are...`